### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Oxford-G**
This project is a simple restaurant homepage, where the task was to build a toggle menu where you can change between the Menu, Home, and Contact tabs.

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] You have mastered Git flow.
- [ ] README includes all the instructions to run the project locally.
- [ ] Project UI is appealing.
## To Improve 🙏
- [ ] Please use ES6 arrow functions instead of function keyword. See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png).
### Happy Coding ☺️
## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

File: src/nav.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import homeFunc from './home';
+import createContact from './contact';
+import createMenu from './menu';
+
+function removeAllChildNodes(parent) {
+  while (parent.firstChild) {
+    parent.removeChild(parent.firstChild);
+  }
+}
```
Comment (oloomoses) [inline src/nav.js]: - [ ] Kindly use the ES6 arrow function instead of the function keyword

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import chesse from './images/chicken.jpg';
+import food from './images/food.jpeg';
+import jollof from './images/jollof.jpeg';
+import veg from './images/veg.jpeg';
+
+function MenuItem(name, src) {
+  this.name = name;
+  this.src = src;
+}
```
Comment (oloomoses) [inline src/menu.js:9]: - [ ] Kindly use the ES6 arrow function instead of the function keyword

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hello @Oxford-G,
**Great job on working on the changes suggested by the previous reviewer,
However, there are few issues to be worked on for the project to be approved**
## To Improve
- [ ] Kindly use the ES6 arrow function instead of the function keyword as suggested in the previous review. Check the comments for details.

**Please make the above changes and ask for another review**

Feel free to leave a comment while mentioning me and I will be happy to reply.
### Happy Coding
_Slack: Oloo Moses_

**Comment by Oxford-G** [comment]
The second function is a constructor function, thereby Es6 cannot be used

**Review by MkrtichSargsyan — APPROVED** [review]
## _Status:_ Approved  :hamster: :rocket:
Hello :wave:.

Awesome job on this project.:rocket: You have completed all the requested changes from the previous code reviewer.

You can go ahead and merge this branch to the base branch of this repo.
Good luck in your future projects and happy coding :relaxed:

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Pull request merged**